frontend/src/app/libs/event/store/event/event.effects.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { Actions, ofType, Effect } from '@ngrx/effects';
+import { catchError, map, exhaustMap } from 'rxjs/operators';
+import { of, Observable } from 'rxjs';
+import { EventActionTypes, LoadEvent, LoadEventSuccess, LoadEventFail } from './event.actions';
+import { Action } from '@ngrx/store';
+import { EventRepository } from '../../services/event-repository.service';
+import { EventDto } from 'src/app/libs/landing/models/event-dto';
+import { EventConverter } from 'src/app/libs/landing/services/event-converter.service';
+
+@Injectable()
+export class EventEffects {
+	@Effect()
+	public GetEvent: Observable<Action> = this.actions$.pipe(
+		ofType(EventActionTypes.LoadEvent),
+		map((action: LoadEvent) => action.payload),
+		exhaustMap((id: string) =>
+			this.eventRepository.getEvent(id).pipe(map((event: EventDto) => new LoadEventSuccess(this.eventConverter.convertFromDto([event])[0])))
```

Comment:
seems like our event converter gets array of event as param, lets refactor it to convert single event.